py3-hdfs.yaml

```diff
@@ -23,8 +23,8 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: 829b659f4ed8f114923690f5b1aa51d295e590bb25eb59ffee00acc1ce493a93
-      uri: https://files.pythonhosted.org/packages/a8/9f/723bf4d4c92e85562430b21b096fb8baa87ff39cd5d784ddd17df2b0146f/hdfs-${{package.version}}.tar.gz
+      expected-sha256: 752a21e43f82197dce43697c73f454ba490838108c73a57a9247efb66d1c0479
+      uri: https://files.pythonhosted.org/packages/29/c7/1be559eb10cb7cac0d26373f18656c8037553619ddd4098e50b04ea8b4ab/hdfs-${{package.version}}.tar.gz
```

Comment:
same as this comment https://github.com/wolfi-dev/os/pull/6936#discussion_r1361716083

it's not needed just will help avoid manual updates in the future WDYT?